specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -30,6 +30,7 @@ export function TreeRow({
   setFocusedRow,
   synonymColor,
   treeName,
+  showApplicableNodes,
```

Comment:
the name `showApplicableNodes` is not very clear as to what it does.

how about `hideLeafNodes` or `hideEmptyNodes` or `hideChildlessNodes`?